### PR TITLE
Remove --pod-cidr from docs

### PR DESCRIPTION
**Pull request by mk46**
Follow-up #31667
/kind cleanup
/language en

### CODE REVIEW

**Comment by mk46** [comment]
@reylejano PTAL

**Comment by reylejano** [comment]
Follow up to PR https://github.com/kubernetes/website/pull/31667

The `--pod-cidr` parameter is deprecated see https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

/lgtm

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ You must use a CNI plugin that is compatible with the
 
 ## Installation
 
-The kubelet has a single default network plugin, and a default network common to the entire cluster.The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
+The kubelet has a single default network plugin, and a default network common to the entire cluster. The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:24]: I'm (still) not sure that “The kubelet has a single default network plugin” remains true after the removal of dockershim.

Comment (reylejano) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:24]:
```suggestion
A CNI plugin is required to implement the [Kubernetes networking model](/docs/concepts/services-networking/#the-kubernetes-network-model). The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
```
I think @sftim is correct. With 1.24 and the removal of dockershim and kubenet then a CNI plugin is required. A technical review is required to confirm. 
Above is only a suggestion but will need a tech review.
/sig node

**Comment by reylejano** [comment]
Cancelling my lgtm until a tech review
/remove lgtm

**Comment by reylejano** [comment]
Cancelling my lgtm until a tech review
/lgtm cancel

**Comment by natalisucks** [comment]
@kubernetes/sig-node-pr-reviews We're hoping for a tech review of the above so we can move ahead with this PR, thanks muchly

**Comment by mehabhalodiya** [comment]
To get reviews from SIG Node people, I have put this PR in the Slack `sig-node` channel to get more attention: https://kubernetes.slack.com/archives/C0BP8PW9G/p1651041175711359

**Review by mikebrow — COMMENTED** [review]
While the point of the PR to remove `--pod-cidr` is valid....

The entire page needs a full tech review and update as the contents are out of date.

**Comment by nate-double-u** [comment]
> While the point of the PR to remove `--pod-cidr` is valid....
> 
> The entire page needs a full tech review and update as the contents are out of date.

Can this PR be merged as is--given its limited scope of removing `--pod-cidr`?

If the rest of the page needs to be updated we should open a new issue to manage that work.

**Comment by mikebrow** [comment]
> > While the point of the PR to remove `--pod-cidr` is valid....
> > The entire page needs a full tech review and update as the contents are out of date.
> 
> Can this PR be merged as is--given its limited scope of removing `--pod-cidr`?
> 
> If the rest of the page needs to be updated we should open a new issue to manage that work.

IMO yes.

**Comment by nate-double-u** [comment]
/approve

Thanks everyone!

**Pull request merged**